### PR TITLE
#32 set default value from schema

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
add new config from panel
add progress indicator when retrieving python informations

### CODE REVIEW

File: l10n/bundle.l10n.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
   "Select property for {0}.": "Select property for {0}.",
   "No {0} entity in toml.": "No {0} entity in toml.",
   "Select {0} entities for {1}.{2}": "Select {0} entities for {1}.{2}",
+  "Retrieving Python informations": "Retrieving Python informations",
```
Comment (FabienLelaquais) [inline l10n/bundle.l10n.json]: Usually, 'informations' in this context is singular... I think

File: l10n/bundle.l10n.json [hunk 2]
```diff
@@ -45,6 +46,11 @@
   "Another {0} element has the identifier {1}": "Another {0} element has the identifier {1}",
   "Using Python interpreter: {0}": "Using Python interpreter: {0}",
   "Select file": "Select file",
+  "File name cannot be empty.": "File name cannot be empty.",
+  "File name {0} already exists.": "File name {0} already exists.",
+  "Cannot create a config file if you don't have at least one workspace folder.": "Cannot create a config file if you don't have at least one workspace folder.",
+  "Enter a new Config file name.": "Enter a new Config file name.",
```
Comment (FabienLelaquais) [inline l10n/bundle.l10n.json:53]: Capital 'Config' here and below, lowercase above.
Please choose.

File: src/schema/validation.ts [hunk 3]
```diff
@@ -164,3 +164,26 @@ export const getPythonReferences = async () => {
   }
   return pythonReferences;
 };
+
+const defaultValues = {} as Record<string, Record<string, string>>;
+export const getDefaultValues = async (nodeType: string) => {
```
Comment (FabienLelaquais) [inline src/schema/validation.ts:169]: Well done!

**Review by FabienLelaquais — APPROVED** [review]
Looks just great!

File: l10n/bundle.l10n.json (hunk 1 above)
Comment (FredLL-Avaiga) [reply l10n/bundle.l10n.json]: OK

**Pull request merged**